From bf8694f6b0b1234fa7cb59611ee66d15000bd5a5 Mon Sep 17 00:00:00 2001
From: Kyle Harding <kyle@balena.io>
Date: Thu, 17 Mar 2022 09:26:31 -0400
Subject: [PATCH 1/1] package/unbound: use system libevent

The file descriptor limit when using the builtin mini-event
cannot handle more than 1024 file descriptors.

https://unbound.docs.nlnetlabs.nl/en/latest/topics/performance.html?highlight=libevent#using-libevent

Signed-off-by: Kyle Harding <kyle@balena.io>
---
 package/unbound/unbound.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package/unbound/unbound.mk b/package/unbound/unbound.mk
index 1e6e0d99d7..782ed2b049 100644
--- a/package/unbound/unbound.mk
+++ b/package/unbound/unbound.mk
@@ -17,6 +17,7 @@ UNBOUND_CONF_OPTS = \
 	--with-pidfile=/var/run/unbound.pid \
 	--with-rootkey-file=/etc/unbound/root.key \
 	--enable-tfo-server \
+	--with-libevent=$(STAGING_DIR)/usr \
 	--with-libexpat=$(STAGING_DIR)/usr \
 	--with-ssl=$(STAGING_DIR)/usr
 
-- 
2.30.2

